New pages for new functions (#221)
NilPointer-Software/bdfd-wiki: src/bdscript/repliedMessageID.md

# $repliedMessageID
Returns the ID of the replied message.

> > 🧙‍♂️ If the message is not a reply, nothing will be returned.

Reviewer: ```suggestion
> 📌 If the message is not a reply, no response will be returned.
```

## Usages
There are two usages of this function :

### Usage #1
#### Syntax
```
$repliedMessageID
```

> Returns the ID of the replied message from the triggering message.

Reviewer: ```suggestion
Returns the ID of the replied message from the author's message.
```

> 

Reviewer: Move the below example to here!

#### Example
```
$nomention

> The message ID you replied to is: $repliedMessageID[$channelID;$messageID]

Reviewer: ```suggestion
The message ID you replied to is: $repliedMessageID
```

```
Returns the ID of the replied message form the given message.

#### Parameters
- `Channel ID` : The channel where the message is located.
- `Message ID` : The message from which the ID of the replied message will be taken from.

> ## Example

Reviewer: ```suggestion
#### Example
```

```
$nomention

> The message ID you replied to is: $repliedMessageID

Reviewer: ```suggestion
The message ID you replied to is: $repliedMessageID[$channelID;$messageID]
```

- `Channel ID` : The channel where the message is located.

> - `Message ID` : The message from which the ID of the replied message will be taken from.

Reviewer: Add an example

# $repliedMessageID

> Returns message ID of the replied message.

Reviewer 2: ```suggestion
Returns the ID of the replied message.
```

> > 🧙‍♂️ If the message is not a reply, nothing is returned.

Reviewer 2: ```suggestion
> 🧙‍♂️ If the message is not a reply, nothing will be returned.
```

## Usages
There are two usages of this function :

### Usage #1
#### Syntax
```
$repliedMessageID
```

> Returns ID of a replied message.

Reviewer 2: ```suggestion
Returns the ID of the replied message from the triggering message.
```

### Usage #2
#### Syntax
```
$repliedMessageID[Channel ID;Message ID]
```

> Returns ID of a replied message.

Reviewer 2: ```suggestion
Returns the ID of the replied message form the given message.
```

> - `Channel ID` : The ID of the channel where the message is.
> - `Message ID` : The ID of the message.

Reviewer 2: ```suggestion
#### Parameters
- `Channel ID` : The channel where the message is located.
- `Message ID` : The message from which the ID of the replied message will be taken from.
```

## Example
```
$nomention
The message ID you replied to is: $repliedMessageID
```
![example](https://user-images.githubusercontent.com/94063167/198900570-ad5b8a25-56ad-4e66-9c3e-6495406d7fda.png)

> 
> 
> 
> 
> 
> 
> 

Reviewer 2: ```suggestion
```
